docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -0,0 +1,80 @@
+Ultraviolet Advanced Oxidation Process
+======================================
+
+Introduction
+------------
+
+Ultraviolet (UV) light is widely used in the water industry to inactivate microorganisms by damaging their nucleic acid, which prevents the pathogen from replicating and causing infection (Wright et al, 2006). Some contaminants resistant to biodegradation, such as N-Nitrosodimethylamine (NDMA), are susceptible to UV disinfection. The high oxidizing capability of UV is often used prior to chemical disinfection to ensure high microbial quality water as well as significantly reducing the chemical dosage required.
+
+Advanced oxidation processes (AOPs) are technologies involving the generation of highly reactive oxidative species, predominantly hydroxyl radicals (:math:`\text{HO} \cdot`). Unlike conventional chemical oxidation processes, such as using chlorine, which are selective as to which compounds they can degrade, AOPs are able to completely convert organic compounds into carbon dioxide, water and mineral acids. In addition, AOPs are feasible for full-scale use to destroy organic compounds because they generate hydroxyl radicals at ambient temperature and atmospheric pressure. Typically, the commercially available UV AOPs for industrial water treatment are:
+
+1) UV light and hydrogen peroxide (:math:`\text{H}_2\text{O}_2`)
+
+2) UV light and ozone (:math:`\text{O}_3`)
+
+3) UV light, hydrogen peroxide (:math:`\text{H}_2\text{O}_2`) and ozone (:math:`\text{O}_3`)
+
+UV AOPs can be  modeled at several different levels, depending on the known kinetic pathways and the modeling objectives. In this work, a basic level kinetic model is presented with an assumption on pseudo-steady state approximation for the kinetic description of free radical species. A pseudo-first order rate constant is utilized to represent the overall degradation rate of contaminants. The users need to provide either UV dose and disinfection rate, or rate constant and exposure time that are can be acquired during UV validation tests. These measurements are then used to simulate the removal of contaminants during the UV AOP process.
+
+This model also accounts for the energy demand for the disinfection process. The users need to provide Electrical Efficiency per Log Order Reduction (EE/O) to evaluate energy costs.
+
+Ports
+-----
+
+The model provides two ports (Pyomo notation in parentheses):
```

Comment:
Would it be helpful to include how you closed the mass balance for a 2-port initial model here? Does the user need to provide an extra degraded component or are there assumptions being made on how the UV 'inactivates' the contaminant?

Reply:
The mass balance for a single input single output unit is simply out = in - loss, similar to `coag_floc_model.rst`. From the paper I reviewed, the UV systems neglect the degraded components and the user also does not need to provide them. I think one reason is that the contaminant concentration handled by UV system is already very low (1e-9 kg/L) from the inlet, and the degraded components have minimal impacts on the water quality. And the degraded components will have a variety of reactions, which depends on the components and can be very challenging to track.